### PR TITLE
CDAP 19685 Feature/Breadcrumb

### PR DESCRIPTION
**Pull request by adityadivami**
# [CDAP-19685] : Modifications to the breadcrumb component of Grid Page to adjust dynamic paths(hyperlinks) to navigate based on the path that the dataset got wrangled.

## Description
Changed breadcrumb paths to display proper navigation path

## PR Type
- [x] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)
https://cdap.atlassian.net/browse/CDAP-19685

## Test Plan
https://docs.google.com/spreadsheets/d/1UCc9Dv1Vi204_sYMGHOlqZ_DujiRYlm01c88CIKmkMk/edit#gid=506288192

## Screenshots

route1
routing2

[CDAP-19685]: https://cdap.atlassian.net/browse/CDAP-19685?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/cdap/components/GridTable/components/Breadcrumb/__tests__/Breadcrumb.test.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+});
+
+const location = {
+  state: {
+    from: 'features.Breadcrumb.params.wrangleHome',
+    path: 'wrangle-home',
+  },
+};
+
+const location2 = {
```
Comment (seanfeld) [inline app/cdap/components/GridTable/components/Breadcrumb/__tests__/Breadcrumb.test.tsx]: is this variable used?

**Review by seanfeld — APPROVED** [review]
just one nitpick with an unused variable?

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const generateDataForExplorationCard = (oldData) => {
           obj.label = `${eachItem[keys]} Recipe steps`;
           obj.type = 'text';
         } else if (keys === 'dataQuality') {
-          obj.label = parseInt(eachItem[keys]);
+          obj.label = parseInt(eachItem[keys], 10);
```
Comment (njbriggs) [inline app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts]: Please use `Number()` to parse numbers in base 10, per https://google.github.io/styleguide/tsguide.html#type-coercion

File: app/cdap/text/text-en.yaml [hunk 3]
```diff
@@ -319,6 +319,14 @@ features:
       message1: "Create a new namespace "
       message2: "(Note: You will require ADMIN privileges for creating a namespace)"
     mainMessage: " You are not authorized to access any existing namespace"
+  Breadcrumb:
```
Comment (njbriggs) [inline app/cdap/text/text-en.yaml]: Per comment in #676, please put all strings under the new header

File: app/cdap/components/GridTable/components/Breadcrumb/__tests__/Breadcrumb.test.tsx (hunk 1 above)
Comment (sivaRajanaDivami) [reply app/cdap/components/GridTable/components/Breadcrumb/__tests__/Breadcrumb.test.tsx]: No. removed this unused variable.

File: app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts (hunk 2 above)
Comment (sivaRajanaDivami) [reply app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts]: Done.

File: app/cdap/text/text-en.yaml (hunk 3 above)
Comment (sivaRajanaDivami) [reply app/cdap/text/text-en.yaml]: Done

**Pull request closed without merging**